src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+import java.util.ArrayList;
+
+/**
+ * Storage class that helps to save or load existing data from local site
+ */
+public class Storage {
+    private final String filePath;
```

Comment:
Shouldn't constant be in uppercase and underline or i think final shouldn't be use if it is not a constant